src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
-
+import Education from './components/Education';
```

Comment:
In order to reduce import statements, create index.js in components/ and export all components there. Then you'll be able to do `import {Education, Experience, General} from './components/'; `